network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/spotlight_posts.html

```diff
@@ -0,0 +1,24 @@
+{% load wagtailimages_tags i18n %}
+
+<div class="row">
+
+  <div class="col-12 col-md-6">
+    <span class="bubble">{% trans "spotlight" %}</span>
```

Comment:
Will the `{% trans "spotlight" %}` text change over time?

Reply:
that's a question for the design team - this is mostly to ensure that it'll get collected for localizing by the pontoon community.